apps/src/templates/projects/ProjectAppTypeArea.jsx

```diff
@@ -76,17 +76,17 @@ class ProjectAppTypeArea extends React.Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
+  componentWillReceiveProps = (nextProps) => {
     this.setState({
       maxNumProjects: nextProps.projectList ? nextProps.projectList.length : 0
     });
-  }
+  };
```

Comment:
react life cycle methods do not need to be bound to `this`, so you can safely leave this method how it was before.